### ISSUE TITLE
Sass-lint process fails if selector has no space between it and opening brace

### ISSUE THREAD

**Issue by micahgodbolt** [sasstools/sass-lint#301]
The following causes sass-lint to fail and return "Warning: Cannot read property 'type' of undefined Use --force to continue."
```
a{
  background: red;
}
```
But this is fine (space after the a tag)
```
a {
  background: red;
}
```

**Comment by DanPurdy** [comment]
There's something really odd happening here...

```scss
a {
  color: red;
}

.test{ <----
  color: red;
}

.test {
  color: red;
}
```

even a block like that in the middle of a file will cause all the other errors and warnings to disappear..

add the space again or change it to the following and all is fine!

**Comment by DanPurdy** [comment]
It's the brace style rule. Looking into it now.

**Comment by DanPurdy** [comment]
@bgriffith you did this rule, fancy looking at this

this line here https://github.com/sasstools/sass-lint/blob/develop/lib/rules/brace-style.js#L49

too late for me to look now but if you use a block as show above there is no last space to check for and so it fails when it starts check for that content, guessing this will be the only case where a space or newline isn't present and thus we can fairly certainly assume that someone has just missed a space before brace and work from there. Might have to provide a default space character though...

**Comment by bgriffith** [comment]
Hey @micahgodbolt this should be fixed by #303. If you have a spare minute it would be greatly appreciated if you could test if this fixes your issue :+1:
